src/Components/GEOSldas_GridComp/GEOSmetforce_GridComp/LDAS_Forcing.F90

```diff
@@ -1898,7 +1928,295 @@ subroutine get_conus_netcdf(date_time, met_path, N_catd, tile_coord, &
+    ! to tile space
+
+    do k=1,N_catd
+       
+       ! ll_lon and ll_lat refer to lower left corner of grid cell
+       ! (as opposed to the grid point in the center of the grid cell)
+       
+       this_lon = tile_coord(k)%com_lon
+       this_lat = tile_coord(k)%com_lat
+       
+       ! add small offsets to avoid unpredictable assignment of
+       ! regularly spaced tiles (such as from the EASEv2 tile space)
+       ! to ERA5 grid cells along certain lat/lon values
+       ! (that is, make it easier to reproduce the mapping done here
+       !  in separate postprocessing analysis scripts)
+       
+       this_lon = this_lon + 0.0001
+       this_lat = this_lat + 0.0001
+       
+       i_ind(k) = ceiling( (this_lon - era5_grid_ll_lon)/era5_grid_dlon )
+       j_ind(k) = ceiling( (this_lat - era5_grid_ll_lat)/era5_grid_dlat )
+
+       ! NOTE: For a "date line on center" grid and (180-dlon/2) < lon < 180 
+       ! we now have i_ind=(grid%N_lon+1) 
+       ! This needs to be fixed as follows:
+       
+       if (i_ind(k)>era5_grid_N_lon)  i_ind(k)=1
+       
+    end do
+    
+    ! read parameters (same for all data variables and time steps)
+
+    ! First read the point2grid (p2g) variable from the mapping.nc file
+    ! 'point2grid' is used to calculate i and j indices for the tmp_grid
+
+    fname = trim(met_path) // '/mapping.nc'
+
+    if(root_logit) write (logunit,*) 'get netcdf params from ' // trim(fname)
+
+    ierr = NF90_OPEN(fname,NF90_NOWRITE,ncid)
+    if (ierr/=0) then
+       err_msg = 'error opening netcdf file'
+       call ldas_abort(LDAS_GENERIC_ERROR, Iam, err_msg)
+    end if
+
+    ! read P2G from the nc file
+    ierr = NF90_INQ_VARID(ncid,'P2G',era5_varid)
+    ierr = NF90_GET_VAR(ncid, era5_varid, p2g)
+    
+    ! calculate x and y indices corresponding to each 1D array element (i.e. tile
+    ! space)
+    do kk=1,N_era5_compressed
+       
+       land_i_era5(kk)=mod((p2g(kk)),era5_grid_N_lon)
+       if(land_i_era5(kk) .eq. 0) land_i_era5(kk)=era5_grid_N_lon
+       
+       land_j_era5(kk)=floor((p2g(kk))/float(era5_grid_N_lon))+1
+       if (mod(p2g(kk),era5_grid_N_lon) .eq. 0) land_j_era5(kk)=land_j_era5(kk)-1
+       
+    end do
+    
+    ! close NC file
+    ierr = NF90_CLOSE(ncid)
+
+    ! Get forcing data geolocation (to be used for Exclude list creation)
+    ! Read in LON and LAT values from only one Forcing file. This information
+    ! is static! Just make sure to use a file where the info is available (since not
+    ! all ERA5 LIS files carry this info)
+
+    fname = trim(met_path) // '/FORCING_201807.nc'
+
+    if(root_logit) write (logunit,*) 'get ERA5 LIS file forcing data lat/lons from ' // trim(fname)
+
+    ierr = NF90_OPEN(fname,NF90_NOWRITE,ncid)
+    if (ierr/=0) then
+       err_msg = 'error opening netcdf file'
+       call ldas_abort(LDAS_GENERIC_ERROR, Iam, err_msg)
+    end if
+
+    ! read Longitude and Latitude from nc file 
```

Comment:
I don't understand why we need the lat/lon values as a gridded variable.  We already know the i,j coordinates for each tile on the 2D ERA5_LIS grid (ie, i_ind, j_ind), and we shouldn't need the lat/lon values to reassemble the forcing data from their compressed 1d array onto the 2d ERA5_LIS grid.  I think the entire section dealing with lat/lon values can be deleted.

Reply:
I removed this section.